src/ray/gcs/accessor.h

```diff
@@ -0,0 +1,182 @@
+#ifndef RAY_GCS_ACCESSOR_H
```

Comment:
I think it's better to just put all the accessors in `gcs_client.h`. Because they are part of `GcsClient`.

Reply:
I think both way is ok.